tests/unit_tests_new/conftest.py

```diff
@@ -0,0 +1,1587 @@
+from collections import defaultdict
```

Comment:
What do you think about the new stuff being:
tests/unit_tests/
and the old stuff being:
tests/unit_tests_old/

so that we can eventually drop the old dir?
is there a reason not to do it this way?

Reply:
The main reason to make it easier for people to merge and also wasn't sure where `tests/unit_tests/` was used, but i see the point of doing it now and just dropping it and avoiding extra renaming... so will do...